store/store.go

```diff
@@ -0,0 +1,127 @@
+package store
+
+import (
+	"context"
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/prometheus/alertmanager/types"
+	"github.com/prometheus/common/model"
+)
+
+var (
+	// ErrNotFound is returned if a Store cannot find the Alert.
+	ErrNotFound = errors.New("alert not found")
+)
+
+// Alerts implements Store using an in-memory map.
+type Alerts struct {
```

Comment:
AlertStore would be more descriptive to me. Alerts sounds like a list of alerts.

Reply:
Except that it will stutter: `store.AlertStore`

Reply:
Yeah, that was the reason it's just `Alerts` :man_shrugging: